Parent/Parent/ParentAppDelegate.swift

```diff
@@ -61,20 +49,21 @@ class ParentAppDelegate: UIResponder, UIApplicationDelegate {
         if hasFirebase {
             FirebaseApp.configure()
         }
-        
-        TheKeymaster.fetchesBranding = false
-        TheKeymaster.delegate = loginConfig
+        NativeLoginManager.shared().setup()
+        NativeLoginManager.shared().app = .parent
```

Comment:
What is this about? Isn't `NativeLoginManager` only needed for React Native?

Reply:
I think it's only for RN, but I didn't have enough confidence to just remove it entirely. I can try that out and see if it still works.